Tutorials/intro-to-timing/readme.md

```diff
@@ -25,29 +25,29 @@ The script presents five examples in order to demonstrate the timing in QUA.
 The examples and the figures of each example are described below:
 
 1. Two pulses from different elements.
-![timing_in_qua](timing_example1.png "Two pulses from different elements")
+![Two_pulses_from_different_elements](timing_example1.PNG "Two_pulses_from_different_elements")
```

Comment:
You don't need a title, and also no need for underscore. I would write this as:
`![Two pulses from different elements](timing_example1.PNG)`

And the same goes for all of the others

Reply:
I saw in intro-to-demod that is was written with underscore so I thought maybe it was a problem as well... 
What is actually the purpose of the sentence written in [ ] ? 
But yes I will just change it and push it...

Reply:
This is the alt text that appear wedged you hover on the image...

Google markdown images, markdown syntax or how to write readme.md github to learn the syntax. It's much faster than trying to guess it...

Reply:
ok, I changed it and did push... what else is missing?

Reply:
nothing, I approved this PR.
You need to resolve this conversation and merge

Reply:
ok :)